README.md

```diff
@@ -54,25 +54,10 @@ This is a list of plugin managers that can help you install these plugins:
 
 ## Archived plugins
 
-The following is a list of archived plugins that are no longer maintained and reside inside the 'archived' subdirectory.
-Any plugins that fail CI will be archived.
+Plugins are archived when they start to fail integration testing with latest CLN release, at which point they will be considered unmaintained.
```

Comment:
```suggestion
If you can't find a plugin you're looking for, it may have been [archived](archived.md). Plugins are archived when they start to fail integration testing with the latest CLN release, at which point they will be considered unmaintained.

If you like a plugin from that list, feel free to update and fix it, so we can un-archive it!
```